src/link/stm32_h750_common.ld

```diff
@@ -111,7 +111,7 @@ SECTIONS
     . = ALIGN(4);
     _ebss = .;         /* define a global symbol at bss end */
     __bss_end__ = _ebss;
-  } >RAM
+  } >D2_RAM
```

Comment:
Should this be reverted?

Reply:
The H7 build won't fit if bss is in RAM as it's only 64K. This really ought to be in DTCM (until such time as we run short of space there and have to start evicting stuff). Unfortunately attempting to put it in DTCM causes a hang on startup, I believe in the bootloader. Working with @hydra to try to fix understand/fix this. It may or may not be specific to SP Racing product. Keen to check this out on another FC.

Reply:
If you can get your hands on a Nucleo H743 board, the MCU on it has got 2 MB RAM, so Betaflight can be run out of RAM with `NUCLEOH743_RAMBASED`, to eliminate the possibility of the boot loader as the culprit.

Reply:
I suggest we revisit this in a another PR once I've got the iFlight FC and have had a chance to investigate further.

Reply:
If i set this back to  ` } >RAM` and modified the `dshot_bitbang.c ` below it will fix #10322